tron/mcp.py

```diff
@@ -20,15 +25,27 @@
 
 
 @contextmanager
-def timer(function_name: str):
-    start = time.time()
+def timer(
+    operation_name: str,
+    histogram_metric: Optional[Histogram] = None,
+    gauge_metric: Optional[Gauge] = None,
+):
```

Comment:
```suggestion
) -> None:
```
non-blocking nit since i'm trying to add types to all of tron :p